Polyhedron/demo/Polyhedron/Scene_polyhedron_item_k_ring_selection.h

```diff
@@ -146,6 +189,33 @@ public Q_SLOTS:
     updateIsTreated();
   }
 
+  // slots are called by signals of surface_mesh_item
+  void sm_vertex_has_been_selected(void* v)
+  {
+    std::size_t h = reinterpret_cast<std::size_t>(v);
+    is_active=true;
+    if(active_handle_type == Active_handle::VERTEX || active_handle_type == Active_handle::PATH)
+      process_selection( static_cast<sm_vertex_descriptor>(h) );
+    updateIsTreated();
+  }
+  void sm_facet_has_been_selected(void* v)
+  {
+    std::size_t h = reinterpret_cast<std::size_t>(v);
+    is_active=true;
+    if (active_handle_type == Active_handle::FACET
+      || active_handle_type == Active_handle::CONNECTED_COMPONENT)
+      process_selection(static_cast<sm_face_descriptor>(h) );
```

Comment:
conversion warning here too : from `size_t` to  CGAL::SM_Index<CGAL::SM_Face_index>::size_type`